feature-libs/product-multi-dimensional/selector/components/product-multi-dimensional-selector-components.module.ts

```diff
@@ -5,9 +5,22 @@
  */
 
 import { NgModule } from '@angular/core';
+import { CmsConfig, provideDefaultConfig } from '@spartacus/core';
 import { ProductMultiDimensionalSelectorModule } from './selector/product-multi-dimensional-selector.module';
+import { ProductMultiDimensionalSelectorComponent } from './selector/product-multi-dimensional-selector.component';
+import { ProductMultiDimensionalSelectorGuard } from './guards';
 
 @NgModule({
   imports: [ProductMultiDimensionalSelectorModule],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ProductMultiDimensionalSelectorComponent: {
```

Comment:
`query:` Just thinking out loud - knowing that the customer's application can't have both options - variants and multi-d, what are the benefits of having separate CMS component for multi-d? Wouldn't be enough to use a variant CMS component?